src/LearnositySdk/Request/Init.php

```diff
@@ -140,6 +142,39 @@ public function __construct($service, $securityPacket, $secret, $requestPacket =
         $this->securityPacket['signature'] = $this->generateSignature();
     }
 
+    /**
+     * Adds metadata to request packet to enable telemetry and tracking. Request packet will be
+     * extended with following parameters:
+     * {
+     *   "meta": {
+     *     "sdk": {
+     *       "version": "v0.10.0",
+     *       "lang": "php",
+     *       "lang_version": "5.6.36",
+     *       "platform": "Linux",
+     *       "platform_version": "#1 SMP Tue Jun 26 16:32:21 UTC 2018"
+     *     }
+     *   }
+     * }
+     *
+     * @param array    $requestPacket
+     *
+     * @return array
+     */
+    private function addMeta(array $requestPacket) {
+        $requestPacket['meta'] = [
+            'sdk' => [
+                'version' => 'v0.10.0', // hardcoded for now
```

Comment:
Could you add the file-reading logic now, and just create the file with, say `git describe > .version` in the root of the SDK code.